### PR TITLE
Update README and change Central properties

### PR DESCRIPTION
**Pull request by io7m**
This adds release instructions to the README.md file, and changes
the existing nexusUsername and nexusPassword properties to use less
confusing mavenCentralUsername and mavenCentralPassword names.

**What's this do?**
Updates documentation.

**Why are we doing this? (w/ JIRA link if applicable)**
Because we have insufficient documentation.

**How should this be tested? / Do these changes have associated tests?**
Not applicable.

**Dependencies for merging? Releasing to production?**
Not applicable.

**Has the application documentation been updated for these changes?**
Yes.

**Did someone actually run this code to verify it works?**
Not applicable.

### CODE REVIEW

**Comment by io7m** [comment]
I also regenerated the module list, because the IDE was telling me that it was out of date.

File: README.md [hunk 1]
```diff
@@ -113,6 +113,127 @@ $ ./gradlew clean assembleRelease test
...
+
+We use [git flow](https://nvie.com/posts/a-successful-git-branching-model/) as our
+basis for branching and creating releases. We highly recommend installing
+[Git Flow AVH Edition](https://github.com/petervanderdoes/gitflow-avh) to
+automate some of the work of branching and tagging. Using `gitflow-avh`
+is not required, but by automating the underlying repository operations,
+it eliminates the possibility of making mistakes, and keeps the various
+branches consistent.
+
+### Releasing
```
Comment (twaddington) [inline README.md]: Having this documented is great! What would you think about moving it into a separate `RELEASING.md` doc and linking to it from here?

**Comment by twaddington** [comment]
Looks great! Left one suggestion to consider.

File: README.md (hunk 1 above)
Comment (io7m) [reply README.md]: Yep, no problem with that. Whatever gets this into a repository somewhere. :wink:

**Comment by io7m** [comment]
If there are no objections, I'll merge this as-is and then move it out into a separate `RELEASING.md` file tomorrow. I can add more information in there regarding PGP keys, the Sonatype web interface, and so on.

**Pull request merged**